<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositWindow</class>
 <widget class="QMainWindow" name="depositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
border-radius: 30px;
background-color : rgb(74, 3, 86);
}
QPushButton pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton:hover {
    background-color : rgb(242, 202, 103);
}</string>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>the amount of charges</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_payment_in_month">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>once a month</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tax_withheld">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color : rgb(232, 143, 197);</string>
    </property>
    <property name="text">
     <string>the period of placement</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_add_one_month">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_deposit_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color : rgb(232, 143, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leave_one_month">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(232, 143, 197);</string>
    </property>
    <property name="text">
     <string>interest rate %</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_kap_1_month">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>once a month</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
border-radius: 30px;
background-color : rgb(195, 28, 139);
}

QPushButton:hover {
    background-color : rgb(102, 104, 104);
}</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>431</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>deposit amount whith interest</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_sub_once_month">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color : rgb(129, 61, 156);</string>
    </property>
    <property name="text">
     <string>withdrawal once a month</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(232, 143, 197);</string>
    </property>
    <property name="text">
     <string>deposit amount</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditcredit_sum_deposit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color : rgb(232, 143, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_percent">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color : rgb(232, 143, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>taxes whithheld</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(129, 61, 156);</string>
    </property>
    <property name="text">
     <string>frequency of payments</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_total_summ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_total_with_tax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>220</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
background-color : rgb(129, 61, 156);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calk_res">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
border-radius: 30px;
background-color : rgb(195, 28, 139);
}

QPushButton:hover {
    background-color : rgb(102, 104, 104);

}</string>
    </property>
    <property name="text">
     <string>calculate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_payment_in_end">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color : rgb(192, 97, 203);</string>
    </property>
    <property name="text">
     <string>end of term</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_add_once_month">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color : rgb(129, 61, 156);</string>
    </property>
    <property name="text">
     <string>replenishment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_percent_tax">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color : rgb(232, 143, 197);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(232, 143, 197);</string>
    </property>
    <property name="text">
     <string>tax rate %</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(129, 61, 156);</string>
    </property>
    <property name="text">
     <string>capitalization %</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>lineEdit_add_one_month</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit_total_summ</zorder>
   <zorder>lineEdit_total_with_tax</zorder>
   <zorder>lineEdit_percent</zorder>
   <zorder>lineEdit_deposit_time</zorder>
   <zorder>lineEditcredit_sum_deposit</zorder>
   <zorder>pushButton_back</zorder>
   <zorder>label_12</zorder>
   <zorder>checkBox_payment_in_month</zorder>
   <zorder>lineEdit_tax_withheld</zorder>
   <zorder>label_9</zorder>
   <zorder>lineEdit_leave_one_month</zorder>
   <zorder>label_3</zorder>
   <zorder>checkBox_kap_1_month</zorder>
   <zorder>pushButton_clear</zorder>
   <zorder>label_13</zorder>
   <zorder>checkBox_sub_once_month</zorder>
   <zorder>label_8</zorder>
   <zorder>label_11</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_calk_res</zorder>
   <zorder>checkBox_payment_in_end</zorder>
   <zorder>checkBox_add_once_month</zorder>
   <zorder>lineEdit_percent_tax</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
